### PR TITLE
Added Transformer and library code test routes

### PR DESCRIPTION
**Pull request by Jayachand**
## Description of the change

> Route transformation code tests in config-backend to rudder-transformer
> Driven by variable TRANSFORMER_TEST_MODE

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by saikumarrs — APPROVED** [review]
LGTM

File: util/customTransformer.js [hunk 1]
```diff
@@ -77,7 +78,14 @@ async function runUserTransform(events, code, eventsMetadata, versionId) {
 
   jail.setSync(
     "_log",
-    new ivm.Reference(() => {
+    new ivm.Reference((...args) => {
```
Comment (sivashanmukh) [inline util/customTransformer.js:81]: Pass testMode into this and enable logs only when testMode = true

File: util/customTransformer-v1.js [hunk 2]
```diff
@@ -43,16 +43,18 @@ function calculateMsFromIvmTime(value) {
...
   /*
  Transform VM aquire mode is
  on demand if env variable ON_DEMAND_ISOLATE_VM = true | True | TRUE,
  Pooled otherwise
 */
-  const isAcquireTransformerIsolatedVMMode = process.env.ON_DEMAND_ISOLATE_VM
-    ? process.env.ON_DEMAND_ISOLATE_VM.toLowerCase() === "true"
-    : false;
+  const isAcquireTransformerIsolatedVMMode =
+    process.env.ON_DEMAND_ISOLATE_VM && !testMode
```
Comment (sivashanmukh) [inline util/customTransformer-v1.js:55]: Fix the condition. Test mode should always avoid pool.

Comment (Jayachand) [reply util/customTransformer-v1.js]: done

File: util/customTransformer.js (hunk 1 above)
Comment (Jayachand) [reply util/customTransformer.js]: done

File: util/ivmFactory.js [hunk 3]
```diff
@@ -118,6 +126,7 @@ async function createIvm(code, libraryVersionIds, versionId) {
       };
     })
   );
+  console.log(compiledModules);
```
Comment (Jayachand) [inline util/ivmFactory.js:129]: delete

**Review by saikumarrs — DISMISSED** [review]

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Pull request merged**